### PR TITLE
build: do not do a full build _after_ we have done an incremental one.

### PR DESCRIPTION
**Pull request by hansl**

### CODE REVIEW

**Comment by jkuri** [comment]
👍  Can you rebase please?
Also, can you merge `_doIncrementalBuild()` and `_doFullBuild()` into one method, seems like some of the code is duplicated?

**Comment by jkuri** [comment]
Or just use `_collectErrors` in the `_doFullBuild()`. If this is something that is going to be done in the future its okay. This fix is actually what we need for now, thanks.

**Comment by hansl** [comment]
Done.

File: lib/broccoli/broccoli-typescript.js [hunk 1]
```diff
@@ -40,14 +40,13 @@ class BroccoliTypeScriptCompiler extends Plugin {
   build() {
     if (!this._tsServiceHost) {
       this._createServiceHost();
-      this._doFullBuild();
-    } else {
+      // this._doFullBuild();
```
Comment (jkuri) [inline lib/broccoli/broccoli-typescript.js]: Is this a note for you for the future work? You could also remove the whole method since its not used :-)

Comment (hansl) [inline lib/broccoli/broccoli-typescript.js]: Yeah that was a test to see if we really needed it, apparently not.

**Comment by jkuri** [comment]
LGTM 👍

**Pull request merged**